src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -95,7 +95,10 @@ export const GettingStarted = () => {
         <EuiHorizontalRule />
         <EuiSpacer />
         <div className="eui-textCenter">
-          <EuiLink onClick={onSkip}>{skipText}</EuiLink>
+          {/* data-test-subj used for FS tracking */}
+          <EuiLink onClick={onSkip} data-test-subj="skipToursLink">
```

Comment:
This is super generic. I'd be a lot more specific so it's easy to just target this single selector and know exactly which tour FS is targeting.


Something like:
```suggestion
          <EuiLink onClick={onSkip} data-test-subj="onboarding--skipUseCaseTourLink">
```